libs/features/personalization/personalization.js

```diff
@@ -142,6 +144,13 @@ const COMMANDS = {
     el.insertAdjacentElement('beforebegin', createFrag(el, target, manifestId));
     el.classList.add(CLASS_EL_DELETE, CLASS_EL_REPLACE);
   },
+  replacemerchcard: (el, cardPath) => {
```

Comment:
I would leave MEP neutral to blocks and just expose the custom verbs/config so that blocks can read and execute.

Reply:
you mean an action agnostic of what block it's amending? do you have an action name in head (same question for @chrischrischris )

Reply:
action name should `replaceMerchCard` but without any implementation.
we can have an array of pull/consume only actions where MEP would simply expose it but do nothing.
:)

Reply:
well, yes, different debate then, (and different JIRA / PR) but i would rather in that case use some "registry" pattern where in that case merch-cards would register its action in mep (cc @chrischrischris)